src/Model.test.tsx

```diff
@@ -0,0 +1,60 @@
+// tslint:disable:no-unused-expression
+
+import * as React from "react";
+
+import {
+  fakeModelModule,
+  initializeTestServices,
+  seedService,
+} from "redux-data-service";
+
+import { spy, stub } from "sinon";
```

Comment:
Imports in import declaration are not used. (spy and stub).
Remove import from sinon.